airflow/config_templates/config.yml

```diff
@@ -1750,7 +1750,14 @@
       default: "IfNotPresent"
     - name: delete_worker_pods
       description: |
-        If True (default), worker pods will be deleted upon termination
+        If True, all worker pods will be deleted upon termination
+      version_added: ~
+      type: string
+      example: ~
+      default: "True"
+    - name: delete_worker_pods_on_success
+      description: |
+        If True (default), worker pods will be deleted only on task success
```

Comment:
You should mention here/for `delete_worker_pods` how the two interact, as it's not clear without reading the code.

Reply:
@ashb oof good point. I'll add that to master and then cherry-pick it in here.